lib/augmentConfig.js

```diff
@@ -250,8 +252,13 @@ function mergeConfigs(a, b) {
 		}
 
 		if (b.overrides) {
-			overridesMerger.overrides = [...new Set(overridesMerger.overrides.concat(b.overrides))];
+			overridesMerger.overrides = overridesMerger.overrides.concat(b.overrides);
 		}
+
+		// Remove duplicates, last one wins
+		overridesMerger.overrides = overridesMerger.overrides.filter(
```

Comment:
question

Overrides are objects, where are you doing the deep comparison to detect duplicates?